### ISSUE TITLE
update MOM6 to GFDL's 20201218 commit

### ISSUE THREAD

**Issue by jiandewang** [ufs-community/ufs-weather-model#341]
## Description
GFDL updated main branch on 20201218, need to make corresponding update in ufs-weather-model
## Solution
replace MOM6 in ufs-weather-model with the updated MOM6 code

## Related to
- this is related to MOM6 PR 45 (https://github.com/NOAA-EMC/MOM6/pull/45#pullrequestreview-555944326) 

**Note**
The updating of MOM6 will not change answer, and will not require any input file updating.

**Comment by jiandewang** [comment]
@junwang-noaa this has been combined with Dom's PR, so can I close this now ?

**Comment by junwang-noaa** [comment]
Let's wait until Dom confirms it.

**Comment by climbfuji** [comment]
> Let's wait until Dom confirms it.
> […](#)

I will make the update in the IPD removal PR (which I am going to take up right now).

**Comment by climbfuji** [comment]
The submodule pointer update to hash 593aecb3be159eca651b727ff3caea2754169118 is now part of https://github.com/ufs-community/ufs-weather-model/pull/331.
```
commit 593aecb3be159eca651b727ff3caea2754169118 (HEAD, origin/dev/emc, origin/HEAD)
Merge: 18bf38869 5d0f8b481
Author: jiandewang <jiande.wang@noaa.gov>
Date:   Sat Dec 19 15:51:32 2020 -0500

    Merge pull request #45 from jiandewang/feature/update-to-GFDL-20201215

    Feature/update to gfdl 20201215
```

**Comment by climbfuji** [comment]
Closed via #331.

**Comment by jiandewang** [comment]
@climbfuji thank you for combining this PR to save us time
